web/assets/css/svg.less

```diff
@@ -1,7 +1,39 @@
 @import (reference) "_vars.less";
 
 svg {
-  .node rect { fill: @grey100; }
+  position: absolute;
```

Comment:
This breaks resource icons on the resource page
<img width="434" alt="Screen Shot 2021-06-04 at 12 01 00 PM" src="https://user-images.githubusercontent.com/26583442/120830535-9725c580-c52c-11eb-9535-d931f64cb4ad.png">

Reply:
Should be fixed now